web/src/input_pill.js

```diff
@@ -363,6 +366,21 @@ export function create(opts) {
             compose.update_on_recipient_change();
         });
 
+        // clicking on an user pill should open up the user's profile
+        store.$parent.on("click", ".pill", function (e) {
+            const participant_user_id = Number.parseInt($(this).attr("data-user-id"), 10);
```

Comment:
It is not clear what is meant by `participant` here. I think having `user_id` as this variable name should be enough.